components/dashboard/src/workspaces/WorkspaceEntry.tsx

```diff
@@ -41,10 +42,21 @@ export function WorkspaceEntry({ desc, model, isAdmin, stopWorkspace }: Props) {
     const downloadURL = new GitpodHostUrl(window.location.href).with({
         pathname: `/workspace-download/get/${ws.id}`
     }).toString();
+    const trackWorkspaceClick = (context:string) => {
+        getGitpodService().server.trackEvent({
+            event:"workspace_open_clicked",
+            properties:{
+                workspace_id: ws.id,
```

Comment:
this is inconsistent with the rest of the system. We've been quite consistent in writing `workspaceId` when it comes to logging and tracing. Also most analytics events already produce `workspaceId`. I'd highly recommend we follow the same naming convention here.